File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/P. First digit !/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/P

#include<iostream>
using namespace std;
 
int main(){
    short x;
    cin >> x;
 
    x/=1000;
    cout << (x&1? "ODD\n": "EVEN\n");
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/P. First digit !/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/P

using System;

namespace Codeforces.Assuit.Sheet1
{
    class ProblemP
    {
        static void Main(string[] args)
        {
            short x = Convert.ToInt16(Console.ReadLine());
            x /= 1000;
            Console.WriteLine(((x&1) == 1? "ODD" : "EVEN"));
        }
    }
}
